From fc2b14fa7f4435a0020b33449362075e05fc4e9f Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Tue, 26 Sep 2023 10:32:01 +0200
Subject: [PATCH 07/38] machines: posix: Fixed an unused parameter warning in
 quit().

The 'n' parameter is not used and triggers a compilation warning.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 machines/posix/tpl_machine_posix.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/machines/posix/tpl_machine_posix.c b/machines/posix/tpl_machine_posix.c
index 4b8709c9..45d7af86 100644
--- a/machines/posix/tpl_machine_posix.c
+++ b/machines/posix/tpl_machine_posix.c
@@ -185,6 +185,8 @@ void tpl_osek_func_stub( tpl_proc_id task_id )
 
 void quit(int n)
 {
+    (void) n;
+
     ShutdownOS(E_OK);
 }
 
-- 
2.34.1

